nengo_viz/static/viz_netgraph_item.js

```diff
@@ -36,7 +36,7 @@ VIZ.NetGraphItem = function(ng, info) {
     this.minHeight = 5;
 
     this.expanded = false;
-    
+
```

Comment:
In tortoisegit you can double click on the file you're committing, which will open the diff and then you can edit things line by line.  I think that's the easiest way to do it.